docker/utils/ports/__init__.py

```diff
@@ -0,0 +1,6 @@
+from .ports import (
+    to_port_range,
+    split_port,
+    add_port_mapping,
+    build_port_bindings
```

Comment:
I don't think `to_port_range` and `add_port_mapping` should be imported here - they're only used within `ports.py`, and I can't see them being useful outside it